src/sonic-config-engine/portconfig.py

```diff
@@ -226,6 +226,16 @@ def get_child_ports(interface, breakout_mode, platform_json_file):
     parent_intf_id = int(re.search("Ethernet(\d+)", interface).group(1))
     for k in match_list:
         offset = gen_port_config(child_ports, parent_intf_id, index, alias_list, lanes, k, offset)
+
+    if hwsku_dict is None:
```

Comment:
As mentioned in https://github.com/Azure/sonic-buildimage/issues/7578 , what we really need is to enhance the design to support other fields in platform.json.